app/iht/utils/AuthHelper.scala

```diff
@@ -16,44 +16,16 @@
 
 package iht.utils
 
-import java.net.{URI, URLEncoder}
+import java.net.URLEncoder
 
-import iht.config.{AppConfig, ApplicationConfig}
-import play.api.mvc.Results._
-import play.api.mvc.{AnyContent, Request}
+import iht.config.ApplicationConfig
 import uk.gov.hmrc.play.config.AppName
-import uk.gov.hmrc.play.frontend.auth._
-import uk.gov.hmrc.play.frontend.auth.connectors.domain.{ConfidenceLevel, CredentialStrength}
-
-import scala.concurrent.Future
 
 object IhtSection extends Enumeration {
   val Registration, Application = Value
 }
 
 trait AuthHelper extends AppName {
-//  private def getCompositePageVisibilityPredicate(postSignInUrl: String, notAuthorisedUrl: String, requiredConfidenceLevel: Int) = {
-//    new CompositePageVisibilityPredicate {
```

Comment:
Not sure why all this was commented and now deleted. Pretty sure they were needed?

Reply:
Had a dig, I see what was done, nvm.